eve/main.yml

```diff
@@ -568,6 +575,10 @@ stages:
         <<: *configure_end2end_tests
         env:
           <<: [*kind-env, *e2e-env, *http-env, *secrets-env]
+    - ShellCommand:
```

Comment:
oh here, sorry forget to mention, as discussed with William, there is also no need to test in sharded mongo. If our tests work in mongo, then they should work in sharded mongo. Sharded mongo is working or not is already tested by other tests.

Reply:
Sure I will remove